docs/content/http-api-authorization.md

```diff
@@ -212,7 +212,7 @@ package httpapi.authz
 # Allow HR members to get anyone's salary.
 allow {
     input.method == "GET"
-    input.path = ["finance", "salary", _]
+    input.path == ["finance", "salary", _]
```

Comment:
Using wildcards (i.e. `_`) sadly doesn't work with `==` at this point (same on line 282). The other changes are good though!

Reply:
I beg to differ. E.g. [here](https://github.com/open-policy-agent/opa/pull/5688/files#diff-401fcb2359d6b142bfa93621679a176c4b628a9b19ae466a2c486dfa0382d21dR54), `input.path == ["finance", "salary", username]` with `username` a fresh variable (introduced with `some` above) will not be safe.

See for example [this playground](https://play.openpolicyagent.org/p/teQ6mQDcDk).

`=` is the unification operator, it's a powerful thing: It takes two sides, left and right, and tries to assign any unset variables to make the two sides _the same_ (i.e. "unify them").

`==` is equality. It's true if both sides _are the same_.

Reply:
Oh, my mistake!